index.html

```diff
@@ -25,24 +26,23 @@
 </head>
 
 <body class="main-block">
-  <header class="head-menu">
-    <nav class="nav-mobile">
-      <a class="logo" href="#">My Logo</a>
-      <a class="logo-ham" href="#"><i class="fa fa-bars"></i></a>
-    </nav>
-    <nav class="nav-desktop">
-      <a class="logo-desktop" href="#">My Logo</a>
-      <div class="nav-element-desktop">
-        <h3 class="elements">Portfolio</h3>
-        <h3 class="elements">About</h3>
-        <h3 class="elements">Contact</h3>
+  
+    <nav class="nav-bar">
+      <div class="nav-mobile">
+        <a class="logo logo-desktop" href="#">My Logo</a>
+        <a class="open-menu" href="#"><i class="fa fa-bars"></i></a>
+        <div class="close-menu hidden"><a href="#"><i class="fa fa-times"></i></a></div>
```

Comment:
Kindly change the ` href ` so when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed and the menu is closed, currently pressing on each link not doing anything.
![image](https://user-images.githubusercontent.com/24703171/156891020-49a75110-1c93-4263-9565-9cdee1dbeed8.png)

Reply:
Thanks for the valuable review. I have fixed the changes.